openapi.yaml

```diff
@@ -55,10 +61,14 @@ info:
           "uuid": "cff8385f-108e-40a5-8f7a-82f3dc0eaba8",
           "metadata": {
             "title": "Some New Title"
+            "revisions": [
+              { "version": "220220517" }
+            ]
           }
         }
       }
       ```
+      In this example, the title is being updated and the version is being added to the list of revisions.
```

Comment:
I think we may want additional emphasis on the _added_ part here. We want to emphasize that no other changes should be made to the `metadata` object as a whole or the contents of the `revisions` list.